apps/gitness/src/framework/context/AppContext.tsx

```diff
@@ -21,21 +23,13 @@ const BASE_URL_PREFIX = `${window.apiUrl || ''}/api/v1`
 const AppContext = createContext<AppContextType | undefined>(undefined)
 
 export const AppProvider: React.FC<{ children: ReactNode }> = ({ children }) => {
```

Comment:
how do we handle casses where the token expires?

Reply:
using an expired token would result in the BE returning a 401 and unsetting the cookie, which we should handle by redirecting user to the signin page.

Reply:
Yes, a `401` logs user out.